src/PKSim.Presentation/DTO/Observers/ObserverDTO.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.DTO;
+using OSPSuite.Utility.Extensions;
+
+namespace PKSim.Presentation.DTO.Observers
+{
+   public class ObserverDTO : ValidatableDTO<IObserverBuilder>
+   {
+      public IObserverBuilder Observer { get; }
+
+      public ObserverDTO(IObserverBuilder observer) : base(observer)
+      {
+         Observer = observer;
+      }
+
+      public IReadOnlyList<string> Details
+      {
+         get
+         {
+            var details = new List<string>();
+            details.Add(InBold("Name"));
+            details.Add(AddNewLine(Observer.Name));
+
+            details.Add(InBold("Formula"));
+            details.Add(AddNewLine(Observer.Formula.ToString()));
+
+            details.Add(InBold("References"));
+            Observer.Formula.ObjectPaths.Each(x => { details.Add($"{x.Alias}:  {x.PathAsString}"); });
+
+            details.Add(StartWithNewLine(InBold("Container Criteria")));
+            details.Add(Observer.ContainerCriteria.ToString());
+            
+            return details;
```

Comment:
Some details are missing here:
* Is imported observer Amount-Observer or Container-Observer
* Is defined for all molecules or not
* Molecules-IncludeList (if defined not for all molecules)
* Molecules-ExcludeList (if defined for all molecules)
* Dimension

Reply:
created an issue here https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1147